crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -50,6 +52,24 @@ use super::ActiveAuthority;
 pub async fn gossip_process<A>(active_authority: &ActiveAuthority<A>, degree: usize)
 where
     A: AuthorityAPI + Send + Sync + 'static + Clone,
+{
+    follower_process(active_authority, degree, GossipDigestHandler::new()).await;
+}
+
+pub async fn node_sync_process<A>(active_authority: &ActiveAuthority<A>, degree: usize)
```

Comment:
Is this not entirely identical to the `gossip_process` above?

Reply:
Yes, it is identical. If we pass in the trait object, `FollowerDigestHandler`, then we don't need two different function signatures.

Reply:
@gdanezis yes, 100% identical. This PR is preparation for another longer PR in which node_sync_process will become different (and way more complicated!)